install/cluster-operator/040-Crd-kafka.yaml

```diff
@@ -696,6 +696,10 @@ spec:
                           the Open Policy Agent, for example, when it is temporarily
                           unavailable). Defaults to `false` - all actions will be
                           denied.
+                      authorizerClass:
```

Comment:
The changes in this file need to go into `packaging/install/cluster-operator/...`

Reply:
I did not follow this suggestion

Reply:
The files in `install/...` should change only during release. The files which need to be changed in a PR like this are in `packaging/...` That is one of the reasons why this needs to be rebased. (more details about it are in https://github.com/strimzi/proposals/blob/master/020-rename-default-branch-of-strimzi-github-repositories.md)